src/pages/Tutorials/Markdown/Universal-Deploy.md

```diff
@@ -0,0 +1,14 @@
+# Universal Deploy
+
+This tutorial guides you through writing a simple flow and deploying it using [Universal Deploy](https://docs.prefect.io/core/PINs/PIN-13-Universal-Deploy.html#pin-13-universal-cloud-deploys), which allows you to register a flow with Prefect Cloud and run it without configuring external storage.
+
+# Set up your local environment
+
+First, let's make sure that your local environment is set up to run Prefect.
+
+Make sure you have a working Python 3.6+ environment with the latest version of Prefect installed. If Prefect is not installed, you can install the latest version from the command line using the package manager of your choice, such as `pip` or `conda`:
```

Comment:
I noticed the renderer interpreted both `pip` and `conda` as full code blocks instead of inline code - any idea why that is happening?

Reply:
After digging into this more, it appears to be a CSS issue. Currently, I'm applying a set of styles to all the `code` elements so it makes it more of a code block instead of inline code.